app/views/welcome/index.html.erb

```diff
@@ -1,5 +1,200 @@
-<div class="text-center">
-  <h1 class="page-header"><%= t 'appname' %></h1>
-  <img class="img-responsive" src="https://bluefin-live-assets.s3.amazonaws.com/media/filer_public_thumbnails/filer_public/30/dc/30dcb5b3-815d-4e4c-af3e-d2823050911c/sports_montage_-_chosen_concept_v9_carouselv3_rgb.jpg__1600x580_q85_crop_upscale.jpg"/>
-  <p class="lead"><%= t '.lead' %></p>
+<!-- Add css file manually, since it does not get linked automatically as it should -->
+<%= stylesheet_link_tag 'welcome' %>
+
+<div class="section darkblue-soccer">
```

Comment:
What´s the advantage of using section over having the class "selection" assigned? As described in https://wiki.selfhtml.org/wiki/HTML/Seitenstrukturierung/section this could lead to unwanted behaviour regarding the headlines.

Reply:
I'm reading about this for the first time now and have never experienced any unwanted behavior with this. Have you tried it in your case?